src/test/daxio/TEST0

```diff
@@ -61,45 +61,45 @@ dd if=/dev/zero bs=1k count=2 2>> prep$UNITTEST_NUM.log | tr '\0' '1' > $DATA2
 dd if=/dev/zero bs=1k count=2 2>> prep$UNITTEST_NUM.log | tr '\0' '2' >> $DATA2
 
 # fill up Device DAX with some random data (len in hex)
-expect_normal_exit $DAXIO$EXESUFFIX -o ${DEVICE_DAX_PATH[0]} -l 16384 < /dev/urandom 2>$LOG
+expect_normal_exit $DAXIO$EXESUFFIX -n -o ${DEVICE_DAX_PATH[0]} -l 16384 < /dev/urandom 2>$LOG
```

Comment:
I think we shouldn't use this option in tests, because it means the environment is misconfigured.

Reply:
Oh, actually we need to mention this in testconfig.sh.example.